Demos/Apps/ThemeApp/ColorTransparencyEdit.razor

```diff
@@ -0,0 +1,16 @@
+﻿<Field Horizontal="true">
+    <FieldLabel ColumnSize="ColumnSize.IsHalf">@Label</FieldLabel>
+    <FieldBody ColumnSize="ColumnSize.IsHalf">
+        <Row>
+            <Column ColumnSize="ColumnSize.Is3">
+                <ColorEdit Color="@Color.Substring(0,7)"
+                           ColorChanged="@(x=>{ Color = x + (Color.Length == 9 ? Color.Substring(7, 2) : string.Empty); })" />
```

Comment:
Hey, if you make an own method for this and give parameter a nice name, it will be prettier than now.

Reply:
I agree. And it would be easier to maintain.

Reply:
Done